### ISSUE TITLE
Sub Commands Issue when Sending from Context.

### ISSUE THREAD

**Issue by jethrob** [Breq16/flask-discord-interactions#31]
When trying to send message from sub command group, receiving and invalid schema error.
It seems like the context (ctx) isn't parsed correctly, the below echo works correctly if using @bp.command()

![image](https://user-images.githubusercontent.com/1051270/124473316-72c53f00-dd9f-11eb-9f39-f26e7e58077a.png)

![image](https://user-images.githubusercontent.com/1051270/124471796-9a1b0c80-dd9d-11eb-8aa5-1f6cdfd4e83b.png)

Sorry I can't send test case, or pull request, still learning to use python.

**Comment by Breq16** [comment]
I was able to reproduce this issue in version 0.1.11, but your example appeared to work fine for me in version 0.1.12. (This is the version that I released after your previous bug report, #29).

You will need to update this library manually, since `pip` does not automatically update packages. The easiest way to do this is with pip's `--upgrade` flag:

```bash
pip3 install Flask-Discord-Interactions --upgrade
```

Let me know if this works for you! 👍
